src/services/victronclient.js

```diff
@@ -10,8 +10,8 @@ const debug = require('debug')('node-red-contrib-victron:victronclient')
  * both subscribe to dbus value updates as well as write values to dbus.
  */
 class VictronClient {
-    constructor(dbusAddress) {
-        this.dbusAddress = dbusAddress || '127.0.0.1'
+    constructor(dbusAddress, dbusPort) {
+        this.dbusAddress = dbusAddress || '127.0.0.1:78'
```

Comment:
Name shadow & confusion: is dbusAddress an ip address or addres + port?

Reply:
Indeed. I think it's better to have separate dbus address and port, but have only one environment variable to set when running node-red. I'll use separate address and port in VictronClient, and split the port and address from the environment variable upon initializing the config node.